client/reader/following/main.jsx

```diff
@@ -8,10 +8,8 @@ import React from 'react';
  */
 import Stream from 'reader/stream';
 
-const FollowingStream = ( props ) => {
-	return (
-		<Stream { ...props } />
-	);
-};
+const FollowingStream = props => (
+	<Stream { ...props } />
+);
```

Comment:
Since this change is actually unrelated to the rest of this PR, I'll revert the commit and file a separate PR to remove the file instead.